### ISSUE TITLE
tensorflowjs_converter does not work for Tensorflow Object Detection API faster-rcnn model

### ISSUE THREAD

**Issue by CPW100** [tensorflow/tfjs#5466]
I trained the **faster_rcnn_resnet50_v1_640x640_coco17_tpu-8 model** using custom dataset using my laptop and exported the model into .pb format as shown below.
![image](https://user-images.githubusercontent.com/28196102/129028169-1e563618-abe0-43cb-8651-5517ddfa9fd9.png)

Then I tried using the 'tensorflowjs_converter' to convert it into .json and bin file. This method works for SSD MobileNet and EfficientDet model but an error pops up whenever I try to convert the faster rcnn .pb model.

This is the **error**:
![image](https://user-images.githubusercontent.com/28196102/129028666-558462f7-ca05-4240-95ee-4c8f68b2851d.png)

Please let me know how can I solve this problem?
Thank you in advance.

**Comment by rthadur** [comment]
As the error message suggests op `broadcast` is not implemented in webgl and wasm , this would be a feature request. Did you try loading in cpu ?

**Comment by rthadur** [comment]
related feature request in wasm https://github.com/tensorflow/tfjs/issues/3580

**Comment by CPW100** [comment]
No, I haven’t tried loading it in CPU, may I know what is the issue with loading it in GPU? How should I proceed?

**Comment by pyu10055** [comment]
We have a PR that is adding the missing BroadcastArgs op.
ref https://github.com/tensorflow/tfjs/pull/4636

**Comment by CPW100** [comment]
Thank you for replying, may I know if the BroadcastArgs op has been added already? If it has already been added, does that mean I need to uninstall tensorflowjs and reinstall it again?

**Comment by pyu10055** [comment]
Not yet, likely will be in the next release. thanks

**Comment by CPW100** [comment]
Alrighty, thanks again. Looking forward to the upgraded version.
